state/factory.go

```diff
@@ -71,7 +71,7 @@ type (
 		GetContractState(hash.PKHash, hash.Hash32B) (hash.Hash32B, error)
 		SetContractState(hash.PKHash, hash.Hash32B, hash.Hash32B) error
 		// Candidate pool
-		candidates() (uint64, []*Candidate)
+		Candidates() (uint64, []*Candidate)
```

Comment:
Need to do this to unblock mockgen. The interface method needs to be public to be visible in another package.

The right fix is to delete this test only interface method